readme.md

```diff
@@ -77,6 +79,7 @@ The most common reason for being stuck on OpenGL 1.1 is non-working GPU drivers
 
 Otherwise:
 * On Windows, you can still run the OpenGL build of ClassiCube anyways. (You can try downloading and using the MESA software renderer from [here](http://download.qt.io/development_releases/prebuilt/llvmpipe/windows/) for slightly better performance though)
+* On Linux,just like Windows,you can either compile a OpenGL 1.1 (seen below) or use MESA´s software renderer LLVMPipe,which is built-in in many Linux distributions,such as Debian,and since LLVMPipe supports OpenGL 4.6,expect it to be slow at mininum and good at maximum.

```

Comment:
Missing spaces between commas, grammatical errors, and unnecessary details about software rendering on Linux, which should be enclosed in brackets and phrased differently.